logstash-core-plugin-api/logstash-core-plugin-api.gemspec

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "logstash-core-plugin-api/version"
+# This is completely valid, when we do a `gem build` the gemspec is 
+# converted to a text file without any ruby code.
+require_relative "../logstash-core/lib/logstash-core/version"
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["Elasticsearch"]
+  gem.email         = ["info@elasticsearch.com"]
```

Comment:
We might changes our other gemspec files? I took the logstash-core.gemspec as the starting point?